Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -25,4 +26,6 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     const double initialStateOfChargePercent_;
 
     // Add your necessary variables here
+    double ampHoursUsed_;
+    BatteryData dataPoint_;
```

Comment:
I think you're actually the first person who's thought of making a BatteryData object, nice!
Rename this so it's a bit more specific, ie it's name should reflect that it's the "Previous" data point.